### PR TITLE
add stroeerCoreBidAdapter

### PR DESCRIPTION
**Pull request by lukashavrlant**

### CODE REVIEW

File: modules/stroeerCoreBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    }
+  },
+
+  interpretResponse: function (serverResponse) {
+    const bids = [];
+
+    if (serverResponse.body && typeof serverResponse.body === 'object') {
+      // TODO: do we need this ajax call?
+      if (serverResponse.body.tep) {
+        ajax(serverResponse.body.tep, () => {
```
Comment (lukashavrlant) [inline modules/stroeerCoreBidAdapter.js]: What about this call? We think we cannot execute ajax calls in this phase. Can we get rid of it? What is the purpose of this call?

File: modules/stroeerCoreBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+    const payload = {
+      id: bidderRequest.auctionId,
+      bids: [],
+      ref: getTopWindowReferrer(),
+      ssl: isSecureWindow(),
+      mpa: isMainPageAccessible(),
+      timeout: bidderRequest.timeout - (Date.now() - bidderRequest.auctionStart),
+      ssat: bidRequestWithSsat ? bidRequestWithSsat.params.ssat : 2, // TODO
+      yl2: bidRequestWithYl2 ? bidRequestWithYl2.params.yl2 : (localStorage.sdgYieldtest === '1'), // TODO
```
Comment (lukashavrlant) [inline modules/stroeerCoreBidAdapter.js]: What about these parameters? (`yl2`, `ssat`, `ab`) Are they required in the official/public Stroeer adapter? Do we need to send them? Do we need to compute the values dynamically, or can we set it to e. g. `2` all the times?

File: modules/stroeerCoreBidAdapter.js (hunk 1 above)
Comment (philipwatson) [reply modules/stroeerCoreBidAdapter.js]: Yes that can be removed. It is used for tracking delivery.

File: modules/stroeerCoreBidAdapter.js (hunk 2 above)
Comment (philipwatson) [reply modules/stroeerCoreBidAdapter.js]: Good spotting. Yes, `yl2`, `ssat` and `ab` can be removed. The backend will use correct defaults.

Also the code related to `bidResponse.bidPriceOptimisation` can be removed.

**Pull request merged**